packages/backend/server/src/core/quota/schema.ts

```diff
@@ -71,6 +71,25 @@ export const Quotas: Quota[] = [
       memberLimit: 3,
     },
   },
+  {
+    feature: QuotaType.FreePlanV1,
+    type: FeatureKind.Quota,
+    version: 3,
+    configs: {
+      // quota name
+      name: 'Free',
+      // single blob limit 10MB
+      blobLimit: 10 * OneMB,
+      // used for business logic
```

Comment:
more clean description about why both `blobLimit` and `bussnessBlobLimit`

Reply:
see here: https://affine-pro.slack.com/archives/C06CF33SK96/p1704702840175909
basically this is a product decision, we will use 100m hard limit in server, but will have smaller limit in frontend based on plan

Reply:
some detail about this field has been add to comment